src/Sql/Sql.Test/ScenarioTests/ElasticJobAgentCrudTests.ps1

```diff
@@ -110,7 +110,7 @@ function Test-UpdateAgent
         Assert-AreEqual $resp.ResourceGroupName $a1.ResourceGroupName
         Assert-AreEqual $resp.Location $a1.Location
         Assert-AreEqual $resp.WorkerCount 100
-        Assert-AreEqual $resp.Tags.Octopus "Agent"
+        # Assert-AreEqual $resp.Tags.Octopus "Agent"
```

Comment:
Is this an intentional change?  Also, does this represent a breaking change?

Reply:
Undid this change